alexmenuet-alexmenuet-taulia/blog/app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,72 @@
+class BlogsController < ApplicationController
+    before_action :authenticate_user!
```

Comment:
The controllers you've created all inherit from `ApplicationsController`. Authenticating a user is something you're going to want across your application for security best practices. You can keep your code DRY by moving `before_action :authenticate_user!` to `ApplicationsController`.